sql/updates/0.11.0--0.11.1-dev.sql

```diff
@@ -33,3 +33,17 @@ CREATE TABLE IF NOT EXISTS _timescaledb_internal.bgw_job_stat (
 
 GRANT SELECT ON _timescaledb_config.bgw_job TO PUBLIC;
 GRANT SELECT ON _timescaledb_internal.bgw_job_stat TO PUBLIC;
+
+DO language plpgsql $$
+BEGIN
+  RAISE WARNING '%',
+ E'\nStarting in v0.12.0, TimescaleDB collects anonymous reports to better understand and assist our
+users. For more information and how to disable, please see our docs https://docs.timescaledb.com/using-timescaledb/telemetry.\n';
+END;
+$$;
+
+CREATE TABLE IF NOT EXISTS _timescaledb_catalog.installation_metadata (
+    key     NAME NOT NULL PRIMARY KEY,
+    value   TEXT NOT NULL
+);
```

Comment:
should we write out a `install_timestamp` here to some well known value indicating "unknown" or "before version 0.12.0" maybe a date like `2001-01-01` or something

Reply:
Alternatively, we could have an `update_timestamp` that we set on each update in, e.g., `pre-update.sql`. If `update_timestamp == install_timestamp` then we know this was a pre 0.12 installation.

Reply:
I think writing `install_timestamp` to some constant value in the update script is enough. Then we don't have to change the endpoint as well.